models/intel/index.md

```diff
@@ -96,7 +96,8 @@ detect areas with complex shape (for example, free space on the road).
 |------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
 | [road-segmentation-adas-0001](./road-segmentation-adas-0001/description/road-segmentation-adas-0001.md)                                        | 4.770                | 0.184      |
 | [semantic-segmentation-adas-0001](./semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md)                            | 58.572               | 6.686      |
-
+| [icnet-camvid-int8-onnx-0001](./icnet-camvid-int8-onnx-0001/description/icnet-camvid-int8-onnx-0001.md)                            | -               | -      |
```

Comment:
maybe do you mean `icnet-camvid-fp32-onnx-0001` and `unet-camvid-fp32-onnx-0001` in this section?
Because CI fails with error:
```
icnet-camvid-fp32-onnx-0001 not in index.md file
```